src/tests/multihost/playbook/build_ssh_package.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: localhost
```

Comment:
Please just add ssh2-python, to https://github.com/SSSD/sssd/blob/master/src/tests/multihost/requirements.txt instead.

Reply:
Done